src/base/utils/version.h

```diff
@@ -167,7 +167,7 @@ namespace Utils
                 throw std::runtime_error ("Incorrect number of version components");
 
             bool ok = false;
-            ComponentsArray res;
+            ComponentsArray res{};
```

Comment:
Without the braces, the array has indeterminate value, like writing `res[100];`.
Now it initialize all elements to `0`.